packages/website/contributors.js

```diff
@@ -26,7 +26,7 @@ http.get(options, function (res) {
 
     res.on('end', function () {
         const contributors = JSON.parse(body);
-        const license = fs.readFileSync('../LICENSE').toString();
+        const license = fs.readFileSync('../../LICENSE').toString();
```

Comment:
This is a nit to prevent path issues:

```js
const fs = require('node:fs');
const path = require('node:path');

const licensePath = path.join(__dirname, '../../LICENSE');
const license = fs.readFileSync(licensePath, 'utf8');
```

What do you think?

Reply:
Done